presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/CachingHiveMetastore.java

```diff
@@ -935,6 +935,12 @@ public void revokeTablePrivileges(MetastoreContext metastoreContext, String data
         return get(tablePrivilegesCache, getCachingKey(metastoreContext, new UserTableKey(principal, databaseName, tableName)));
     }
 
+    @Override
+    public void setPartitionLeases(MetastoreContext metastoreContext, String databaseName, String tableName, Map<String, String> partitionNameToLocation, int leaseDurationSeconds)
```

Comment:
We prefer `io.airlift.units.Duration` object to express duration. I recommend changing `int leaseDurationSeconds` to `Duration leaseDuration`.